cypress-tests/cypress/e2e/ConnectorUtils/Commons.js

```diff
@@ -0,0 +1,482 @@
+// This file is the default. To override, add to connector.js
+import State from "../../utils/State";
+
+const globalState = new State({
+  connectorId: Cypress.env("CONNECTOR"),
+  baseUrl: Cypress.env("BASEURL"),
+  adminApiKey: Cypress.env("ADMINAPIKEY"),
+  connectorAuthFilePath: Cypress.env("CONNECTOR_AUTH_FILE_PATH"),
+});
+
+const connectorId = globalState.get("connectorId");
+
+const successfulNo3DSCardDetails = {
+  card_number: "4111111111111111",
+  card_exp_month: "08",
+  card_exp_year: "25",
+  card_holder_name: "joseph Doe",
+  card_cvc: "999",
+};
+
+const successfulThreeDSTestCardDetails = {
+  card_number: "4111111111111111",
+  card_exp_month: "10",
+  card_exp_year: "25",
+  card_holder_name: "morino",
+  card_cvc: "999",
+};
+
+// Function to get default PaymentIntent object
+const getDefaultIntent = () => ({
+  Request: {
+    currency: "EUR",
+  },
+  Response: {
+    status: 501,
+    body: {
+      error: {
+        type: "invalid_request",
+        message: `Selected payment method through ${connectorId} is not implemented`,
```

Comment:
In intent it won't throw an error(Always it will be success), In the confirm call it will throw an error. So can you make the change accordingly